kind-istio/kind-config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
Is this copied/inspired from somewhere? If that is the case please drop a line with the link

Reply:
It has been quite a lot revisited, but I added both the doc and file I used to do it